src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -214,38 +214,56 @@
+                                        stimulus_action('araise/core-bundle/dropdown', 'layoutCalculate', 'resize@window') }}
+                                        type="button"
+                                    >
+                                        <span class="sr-only">Optionen</span>
+                                        {{ bootstrap_icon('three-dots-vertical', { class: 'h-4 w-4' }) }}
+                                    </button>
+
+                                    <div
+                                        {{ stimulus_target('araise/core-bundle/dropdown', 'menu') }}
+                                        class="z-50 hidden transition duration-300 origin-top-right mx-3 origin-top-right absolute right-7 top-0 -translate-x-1 w-48 -mt-1 rounded-md shadow-lg z-10 bg-white ring-1 ring-black ring-opacity-5 divide-y divide-neutral-200 focus:outline-none"
+                                        data-transition-enter-from="opacity-0 scale-95"
+                                        data-transition-enter-to="opacity-100 scale-100"
+                                        data-transition-leave-from="opacity-100 scale-100"
+                                        data-transition-leave-to="opacity-0 scale-95"
+                                        tabindex="-1"
+                                    >
+                                        <div class="py-1" role="none">
+                                            {% for action in tableActionsVisible %}
+                                                <span class="block">
+                                                    {{ araise_table_action_render(action, row) }}
+                                                </span>
+                                            {% endfor %}
+                                        </div>
                                     </div>
                                 </div>
-                            </div>
+                            {% else %}
+                                {% set action = tableActionsVisible|first %}
+                                <a
+                                    href="{{ path(action.route, action.routeParameters(row)) }}"
+                                    class="whatwedo_table-actions-button"
+                                    {{ action.option('attr')|map((value, attr) => "#{attr}=\"#{value}\"")|join(' ')|raw }}
+                                >
+                                    <span {{ stimulus_controller('araise/crud-bundle/tooltip', { 'title': action.label|trans } ) }}>
+                                        {% if action.option('icon') %}
+                                            {{ bootstrap_icon(action.icon, { class: 'h-4 w-4' }) }}
+                                        {% else %}
+                                            {{ bootstrap_icon('link-45deg', { class: 'h-4 w-4' }) }}
+                                        {% endif %}
+                                    </span>
```

Comment:
It looks like there is an accessible text missing for this button. Maybe the tooltip component is adding it, I can't tell, but at best this is not part of the progressive enhancement and placed as a `<span class="sr-only">Action Text</span>`.